src/rachanaGJun20/String/PhoneNumberValidation.java

```diff
@@ -0,0 +1,69 @@
+/*Assignment 24 :- 5th Jul'2020 
+Validate given phone number.
+
++91 9765463742 --> valid
+09765463742 --> valid
+976 546 3742 --> valid
+9765463742 --> valid
+
+9765463A42 --> Invalid
+976546342 --> Invalid
+6976546342 --> Invalid
+
+Note : phone number is given in String format.*/
+package rachanaGJun20.String;
+
+public class PhoneNumberValidation {
+	// method will validate given number for prefix
+	void validatePhNumber(String phoneNumber) {
+
+		phoneNumber = phoneNumber.replaceAll(" ", "");
+
+		if (phoneNumber.startsWith("+91")) {
+			phoneNumber = phoneNumber.substring(3);
+			validNumber(phoneNumber);
+		} else if (phoneNumber.startsWith("0")) {
+			phoneNumber = phoneNumber.substring(1);
+			validNumber(phoneNumber);
+		} else
+			validNumber(phoneNumber);
+	}
+
+	// method will perform required validations on given phone number
+	void validNumber(String phoneNumber) {
+		if (phoneNumber.length() == 10) {
+			if (phoneNumber.startsWith("9") || phoneNumber.startsWith("8") || phoneNumber.startsWith("7")) {
+				if (isAllDigits(phoneNumber)) {
+					System.out.println("Number is Valid");
+				} else
+					System.out.println("Number not valid");
+			} else
+				System.out.println("Number not valid");
+		} else
+			System.out.println("Number not valid");
+	}
+
+	// it will check that number does not contains any character
+	boolean isAllDigits(String phoneNumber) {
+
+		boolean flag = true;
+		for (int index = 1; index < phoneNumber.length(); index++) {
+			if (!Character.isDigit(phoneNumber.charAt(index))) {
+				flag = false;
+			}
+		}
+		if (flag)
+			return true;
+		else
+			return false;
+	}
+	
+	public static void main(String[] args) {
+		PhoneNumberValidation phonenumbervalidation = new PhoneNumberValidation();
+		String[] phNumber = { "+91 9765463742", "09765463742", "976 546 3742", "9765463742", "9765463A42", "976546342",
```

Comment:
Can you check with input like this
+91 9765 463742
9765 46 3742
@RachanaGJun20   length and spacing of phonenumber not handled properly. 
  
@KrishnaKTechnocredits Plzz give your input.

Reply:
![image](https://user-images.githubusercontent.com/66841434/86807138-36c9a200-c097-11ea-8158-5ae6a340bbdd.png)


@AjitSJune20  Hey Ajit, Thanks for giving the another input numbers, my program is working for your given combination as well. Please have a look.

Reply:
It should throw Invalid @RachanaGJun20 bcoz of exceeding length of PhoneNumber and space is at wrong place in Phinenumber